### DISCUSSION TITLE
Filter Params to DBT

### DISCUSSION THREAD

**Discussion by Nopalin** [lightdash/lightdash#10832] [Q&A]
I am a fan of optimization and simplicity (as everybody here) so i try to design our data model to not need extra tables just for reporting, the tables in the system logic are used to extract data in real time. So all of our query attemt to be as concise and fast as posible, and the way we achieve that is by sending to the DB engine the exact query we need.
So in order to do that we define a dbt model with a predefined date filter, which i'd like to set in lightdash when i want to use that model, but i could not found how to achieve that because the predefined filter are over a column that are not defined as dimension, so, the question is, can i set those filters in lightdash? i just found a very little sql parameters that are passed to dbt (one of them are user/group attributes which let me say are marvelous function) but another kind of dtb parameter seems no to be allowed, this is correct? or could it be that i havent been capable of finding that function?

Thanks

**Comment by jpetey75** [comment]
@Nopalin can you share the documentation for the dbt feature you're using in that model? I _think_ what you need might be this: https://github.com/lightdash/lightdash/issues/4172 but I'm not sure I understand your question yet.
